pkg/ingester/metrics.go

```diff
@@ -496,6 +512,29 @@ func newTSDBMetrics(r prometheus.Registerer) *tsdbMetrics {
 			"Total number of TSDB checkpoint creations attempted.",
 			nil, nil),
 
+		tsdbExemplarsTotal: prometheus.NewDesc(
+			"cortex_ingester_tsdb_exemplars_appended_total",
```

Comment:
Please name these metrics exactly like Prometheus but with replacing the `prometheus_` prefix with `cortex_ingester_`. Following this pattern, even if the naming is not beautiful, this metric should be `cortex_ingester_tsdb_exemplar_exemplars_appended_total`.

_Same comment applies to other new metrics below._